src/aero/core.clj

```diff
@@ -34,7 +34,8 @@
          config
          (with-open [pr (java.io.PushbackReader. (io/reader r))]
            (edn/read
-            {:readers (readers {:profile (or profile :default)
+            {:eof {}
```

Comment:
I was thinking along the empty file-empty config versus no file-no config lines (although no file will still throw and that might be desired). I see your point and would have no trouble changing it to nil if you prefer that.